features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -614,6 +614,17 @@
         Default value : false
         -->
         <!--<DropUnregisteredScopes>true</DropUnregisteredScopes>-->
+
+        <!--
+            This config will remove any optional claims from introspection response. Before setting the response to the
+            introspection endpoint claims configured below will be removed.
+        -->
+        <!--
+        <IntrospectionResponse>
+            <UnsupportedClaim></UnsupportedClaim>
```

Comment:
UnsupportedClaim might give the wrong impression.
How about using something like filtered claim?

Also, i feel the XML structure should be something like below. Otherwise we won't be able to add any configurations related to introspection in future.

<IntrospectionResponse>
   <FilteredClaims>
            <FilteredClaim></FilteredClaims>


Also we already have a configuration section <Introspection> . Can't we have this config under that?
            
@pulasthi7 @janakamarasena WDYT?

Reply:
Changed in https://github.com/wso2/carbon-identity-framework/pull/3782/commits/969678207b42123443e2727907ae1434297f2215